model/registry/artifact.yaml

```
@@ -0,0 +1,95 @@
+groups:
+  - id: registry.artifact
+    prefix: artifact
+    type: attribute_group
+    brief: >
+      This group describes attributes specific to artifacts. Artifacts are
+      files or other immutable objects that are intended for distribution. This
+      definition aligns directly with the
+      [SLSA](https://slsa.dev/spec/v1.0/terminology#package-model) package
+      model.
+    attributes:
+      - id: filename
+        type: string
+        stability: experimental
+        brief: >
+          The human readable file name of the artifact, generated by
+          build and release utilities to include the package name and version.
+        note: |
+          This file name can also act as the [Package Name](https://slsa.dev/spec/v1.0/terminology#package-model)
+          in cases where the package ecosystem maps accordingly.
+          Additionally, the artifact [can be published](https://slsa.dev/spec/v1.0/terminology#software-supply-chain)
+          for others, but that is not a guarantee.
+        examples:
+          [
+            "golang-binary-amd64-v0.1.0",
+            "docker-image-amd64-v0.1.0",
+            "release-1.tar.gz",
+            "file-name-package.tar.gz",
+          ]
+      - id: version
+        type: string
+        stability: experimental
+        brief: >
+          The version of the artifact.
+        examples: ["v0.1.0", "1.2.1", "122691-build"]
+      - id: purl
+        type: string
+        stability: experimental
+        brief: >
+          The [Package URL](https://github.com/package-url/purl-spec) of the
+          [package artifact](https://slsa.dev/spec/v1.0/terminology#package-model),
+          providing a standard way to identify and locate the packaged artifact.
```

nit: using simple sentence feels a bit straightforward here. But feel free to decline
```suggestion
          The [Package URL](https://github.com/package-url/purl-spec) of the
          [package artifact](https://slsa.dev/spec/v1.0/terminology#package-model)
          provides a standard way to identify and locate the packaged artifact.
```

---

I agree, read better. Made adjustment!